src/types.js

```diff
@@ -42,5 +130,17 @@ export const TYPES = {
   lawEnforcementGuidelines: {
     name: 'Law Enforcement Guidelines',
     fileName: 'law_enforcement_guidelines'
-  }
+  },
+  consumerTermsOfSale: {
+    name: 'Consumer Terms of Sale',
+    fileName: 'consumer_terms_of_sale'
+  },
+  securityAndPrivacy: {
```

Comment:
In favor of merging with `privacy`, unless I misunderstand and there is a difference.
Could we have an example?